lib/xebow_web/templates/layout/root.html.leex

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <%= csrf_meta_tag() %>
+    <%= live_title_tag assigns[:page_title] || "RGBMatrix", suffix: " · Phoenix Framework" %>
```

Comment:
:warning: (recommended)

Considering this is the title of the application/product, how about removing `Phoenix Framework` from the page title and changing the suffix to `- Xebow`? (i.e. display a title like `Dashboard - Xebow`)

Or use a flashy product name for the UI like `Xebow Studio`. :grin: